src/pages/MetricDetail.svelte

```diff
@@ -121,46 +128,53 @@
 
   <h2>Access</h2>
 
-  <table>
-    <col />
-    <col />
-    <tr>
-      <td>
-        BigQuery
-        <HelpHoverable content={'The BigQuery repesentation of this metric.'} />
-      </td>
-      <td>
-        {#each metric.bigquery_names.stable_ping_table_names as [sendInPing, tableName]}
-          <div>
-            In
-            <a
-              href={getMetricBigQueryURL(params.app, sendInPing)}>{tableName}</a>
-            <!-- Skip search string for event metrics as we can't directly lookup the columns in events tables -->
-            {#if metric.bigquery_names.metric_type !== 'event'}
-              as
-              <a
-                href={getMetricBigQueryURL(params.app, sendInPing, metric.bigquery_names.metric_table_name)}>
-                {metric.bigquery_names.metric_table_name}
-              </a>
-            {/if}
-          </div>
-        {/each}
-      </td>
-    </tr>
-    {#if glamUrl && metric.bigquery_names.metric_type !== 'event'}
+  {#if metric.variants.length > 1}
+    <AppVariantSelector bind:selectedAppVariant variants={metric.variants} />
+  {/if}
+
+  {#if selectedAppVariant}
+    <table>
+      <col />
+      <col />
       <tr>
         <td>
-          GLAM
+          BigQuery
           <HelpHoverable
-            content={'View this metric in Glean Aggregated Metrics'} />
+            content={'The BigQuery representation of this metric.'} />
         </td>
         <td>
-          <a
-            href={glamUrl(metric.bigquery_names.glam_etl_name)}>{metric.bigquery_names.glam_etl_name}</a>
+          {#each selectedAppVariant.bigquery_names.stable_ping_table_names as [sendInPing, tableName]}
+            <div>
+              In
+              <a
+                href={getMetricBigQueryURL(params.app, selectedAppVariant.app_id, sendInPing)}>{tableName}</a>
+              <!-- Skip search string for event metrics as we can't directly lookup the columns in events tables -->
+              {#if selectedAppVariant.bigquery_names.metric_type !== 'event'}
+                as
+                <a
+                  href={getMetricBigQueryURL(params.app, selectedAppVariant.app_id, sendInPing, selectedAppVariant.bigquery_names.metric_table_name)}>
+                  {selectedAppVariant.bigquery_names.metric_table_name}
+                </a>
```

Comment:
This link now returns a Not Found error.

<img width="886" alt="CleanShot 2021-03-01 at 14 03 20@2x" src="https://user-images.githubusercontent.com/28797553/109565190-f2ac6700-7a96-11eb-945e-195518da0cfa.png">